about.html

```diff
@@ -0,0 +1,173 @@
+  <!-- css -->
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- javascript -->
+  <script src="./javascript/navbar.js" defer></script>
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <div class="socials">
+      <ul>
+        <li>
+          <i class="fa-brands fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li>
+          English
+        </li>
+        <li>
+          My Page
+        </li>
+        <li>
+          Logout
+        </li>
+      </ul>
+    </div>
+    <nav>
+      <ul class="navbar">
+        <li class="logo">
+          <a href="index.html">
+            <img src="./images/blockchain-logo.png" alt="Logo" class="navbar-logo">
+          </a>
+        </li>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li>
+          <a href="#">Sponsor</a>
+        </li>
+        <li>
+          <a href="#">News</a>
+        </li>
+        <li>
+          <a href="#">
+            <button class="join-button" type="button">Block Summit</button>
+          </a>
+        </li>
+      </ul>
+      <div class="hamburger-menu-icon">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="about">
+      <div id="about-intro">
+        <div id="hero-about">
+          <p class="hero-greeting">
+            "Hello! Blockchain Community"
+          </p>
+          <h1 id="hero-heading">
+            BLOCKCHAIN ON-RAMP GLOBAL SUMMIT 2023
+          </h1>
+        </div>
+        <div id="hero-about-description">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Error consequuntur enim hic ratione soluta mollitia ipsam aperiam tempora assumenda molestias consectetur nobis, velit facilis quam commodi illum obcaecati dolores nesciunt maiores ex blanditiis quia quas, voluptas culpa! Id, accusantium! Praesentium omnis molestiae in deserunt nisi blanditiis velit saepe placeat? Facilis.
+        </div>
+        <div id="contact-call">
+          <p>
+            Please contact us per Email for any further questions about Blockchin On-ramp Global Summit 2023!
+          </p>
+          <a href="mailto:njugunab655@gmail.com">njugunab655@gmail.com</a>
+        </div>
+      </div>
+      <div class="white">
+        <div id="about-logo">
+          <h2>Blockchain On-ramp Global Summit 2023 Logo</h2>
+          <hr class="section-heading-underline">
+          <p>The logo for Blockchain On-ramp Global Summit 2023 was decided during an art competition held from 14th February to 28th February.</p>
```

Comment:
- [ ] While you do a great effort 😃 it will be better if you increase the padding of the text here to make it like the Behance design, so please 😊 try to increase the padding to be like the Behance 💯
- [ ] Kindly 😃 try to stick to the Behance style, so please try to give the text here the same color of the text to be like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225304669-4ec5b8e8-df40-421d-bcb9-3ebaccce8b4e.png)  |    ![image](https://user-images.githubusercontent.com/83360636/225304601-d496bbf4-3903-40cd-968c-191256f456e5.png)  |